.github/workflows/best-practice.yml

```diff
@@ -1,27 +1,30 @@
 # This workflow will build a Java project with Maven
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
 
-name: Front-end performance
+name: Best Practices Tests
 
 on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
+    strategy:
+      fail-fast: false
+      matrix:
+        test: [ DesktopTests, EmuSimWeb*, Visual*]
```

Comment:
@titusfortner I see that you removed front-end perf testing. Hence, now we wont' have a badge to tell us if it's working or not. Curious why you decided on this?

Reply:
The test is in the JUnit.4 code, so it is being run with the other JUnit 4 tests. I was trying to keep the tests and the modules in synch, but the one outlier is anything that uses a Real Device since you can't run too many at a time without failures

Reply:
Oops, I read the code wrong, you are3 right, it got pulled out. Also we don't need to matrix this; tests are small enough we can run them all in parallel: https://github.com/saucelabs-training/demo-java/pull/110/files#diff-0a944e8c9fa2dbc36f9e5dc16dd929ad3fb3049cfc30d4113d825bd93965871dR26